x/crisis/internal/keeper/handler.go

```diff
@@ -1,22 +1,22 @@
-package crisis
+package keeper
```

Comment:
uhhhhh why is the handler part of the keeper pkg ? It should live on the root of the module

Reply:
I don't feel good about it either - it's there to avoid import cycles.
I've done it anyway because from users perspective it has no impact

Reply:
@alessio please review the module spec [here](https://github.com/cosmos/cosmos-sdk/blob/master/docs/modules/SPEC.md).

This should remain as-is.